packages/common/test/content.test.ts

```diff
@@ -961,12 +964,57 @@ describe("getAdditionalResources", () => {
         },
       },
     };
-    const result = getAdditionalResources(item, metadata);
+    const result = extractRawResources(metadata);
+    expect(result.length).toEqual(1);
+    expect(result[0]).toEqual({
+      orName: "Resource Name",
+      linkage: "resource-url",
+    });
+  });
+
+  it("handles when distTranOps is an array", () => {
+    const metadata = {
+      metadata: {
+        distInfo: {
+          distTranOps: [
+            {
+              onLineSrc: {
+                orName: "Resource Name",
+                linkage: "resource-url",
+              },
+            },
+          ],
+        },
+      },
+    };
+    const result = extractRawResources(metadata);
+    expect(result.length).toEqual(1);
+    expect(result[0]).toEqual({
+      orName: "Resource Name",
+      linkage: "resource-url",
+    });
+  });
+
+  it("handles when distInfo is an Array", () => {
```

Comment:
if other combinations are not covered by tests, I would put them here. Namely the matrix of:
- distInfo as object, array of 1, or array of 2
- distTransOps as object, array of 1, or array of 2
- onLineSrc as object, array of 1, array of 2